bower.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "adapt-svg",
-  "version": "2.1.0",
+  "version": "3.0.1",
   "framework": ">=5",
```

Comment:
may as well bump FW to 5.3 and then you can use ES6

Reply:
I went straight for 5.8